src/main/java/com/brightcove/zencoder/client/model/State.java

```diff
@@ -21,6 +21,7 @@
     FAILED("failed"),
     CANCELLED("cancelled"),
     NO_INPUT("no_input"),
+    UPLOADING("uploading"),
```

Comment:
Issue #10 prompted this change; customer reported encountering an exception while parsing responses for outputs that were in an 'uploading' state.